### PR TITLE
Update story-issue-template-.md

### PR DESCRIPTION
**Pull request by jadudm**
Per work yesterday.

* Improved given/when/then
* Added shepherds
* Added shepherds to checklist
* Hid the checklist under a flippy triangle.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/story-issue-template-.md [hunk 1]
```diff
@@ -1,38 +1,65 @@
...
+
+### Scenario: 
+
+**Given**   
+**when**  
+...
+
+```[tasklist]
+### then... 
+- [ ]
```
Comment (mogul) [inline .github/ISSUE_TEMPLATE/story-issue-template-.md:33]:
```suggestion
- [ ] [a verifiable outcome]
```

File: .github/ISSUE_TEMPLATE/story-issue-template-.md [hunk 2]
```diff
@@ -1,38 +1,65 @@
...
-[Any helpful contextual notes or links to artifacts/evidence, if needed]
+[comment]: # "Any helpful contextual notes or links to artifacts/evidence, if needed"
 
----
+# Security Considerations
 
-# Security Considerations ([required](https://nvd.nist.gov/800-53/Rev4/control/CM-4))
+Required per [CM-4](https://nvd.nist.gov/800-53/Rev4/control/CM-4).
 
 [comment]: # "Our SSP says 'The Data.gov team ensures security implications are considered as part of the agile requirements refinement process by including a section in the issue template used as a basis for new work.' so please don't remove this section without care."
```
Comment (mogul) [inline .github/ISSUE_TEMPLATE/story-issue-template-.md:51]: Whoops, this shouldn't have been brought over. Removing the specific name of the team for better bias toward reuse in the project template later, once things are rolling smoothly...
```suggestion
[comment]: # "Our SSP says 'The team ensures security implications are considered as part of the agile requirements refinement process by including a section in the issue template used as a basis for new work.' so please don't remove this section without care."
```

File: .github/ISSUE_TEMPLATE/story-issue-template-.md [hunk 3]
```diff
@@ -1,38 +1,65 @@
...
+[comment]: # "Begin with a short summary of this ticket, so its intent can be understood at a glance."
+
 
 # Acceptance Criteria
 
-[ACs should be clearly demoable/verifiable whenever possible. Try specifying them using [BDD](https://en.wikipedia.org/wiki/Behavior-driven_development#Behavioral_specifications).]
+We use [behavior-driven development](https://en.wikipedia.org/wiki/Behavior-driven_development#Behavioral_specifications) wherever possible.
+
+[comment]: # "ACs should be clearly demoable/verifiable whenever possible. Try specifying them using behavior driven development."
+[comment]: # "**Given** **when** **then**"
```
Comment (mogul) [inline .github/ISSUE_TEMPLATE/story-issue-template-.md:19]: This line isn't adding much.

```suggestion
```

File: .github/ISSUE_TEMPLATE/story-issue-template-.md [hunk 4]
```diff
@@ -1,38 +1,65 @@
...
+[comment]: # "Given: the initial context at the beginning of the scenario"
+[comment]: # "when: the event that triggers the scenario"
+[comment]: # "then: the expected outcome(s)"
+[comment]: # "Repeat scenarios as needed, or repeat behaviors and lists within a scenario as needed."
+
+### Scenario:
```
Comment (mogul) [inline .github/ISSUE_TEMPLATE/story-issue-template-.md:25]:
```suggestion
### Scenario: [plain language description]
```

**Review by mogul — APPROVED** [review]
A great improvement! 

These suggestions are all optional (hence not blocking my approval), but I'd prefer to see them in. (I want to encourage convergence toward [the Dan North template](https://en.wikipedia.org/wiki/Behavior-driven_development#Behavioral_specifications) which does a good job of pairing the typical story statement with a series of BDD scenarios/ACs.)

**Comment by mogul** [comment]
Side note: I mentioned in passing that when writing several BDD scenarios that are very similar (eg only slight variations in the people/actions/outcomes between them) there is [a good-practice syntax for applying the Don't Repeat Yourself (DRY) principle, using backgrounds, outlines, and tables](https://docs.behat.org/en/latest/user_guide/writing_scenarios.html#backgrounds). (Though this example comes from the Behat documentation, it's not actually tool-specific, and many Gherkin-friendly BDD tools accept something nearly identical, if we ever get as far as wanting to incorporate one).

File: .github/ISSUE_TEMPLATE/story-issue-template-.md [hunk 5]
```diff
@@ -1,38 +1,65 @@
...
+
+[comment]: # "Begin with a short summary of this ticket, so its intent can be understood at a glance."
+
 
 # Acceptance Criteria
 
-[ACs should be clearly demoable/verifiable whenever possible. Try specifying them using [BDD](https://en.wikipedia.org/wiki/Behavior-driven_development#Behavioral_specifications).]
+We use [behavior-driven development](https://en.wikipedia.org/wiki/Behavior-driven_development#Behavioral_specifications) wherever possible.
+
+[comment]: # "ACs should be clearly demoable/verifiable whenever possible. Try specifying them using behavior driven development."
```
[the remark this answers is not in the document]
Comment (jadudm) [reply .github/ISSUE_TEMPLATE/story-issue-template-.md]: Cleaned up.

File: .github/ISSUE_TEMPLATE/story-issue-template-.md [hunk 6]
```diff
@@ -1,38 +1,65 @@
...
+title: 'clear, concise summary'
 labels: ''
 assignees: ''
 ---
-# User Story
 
-In order to [goal], [stakeholder] wants [change].
+# At a glance
+
+[comment]: # "Begin with a short summary of this ticket, so its intent can be understood at a glance."
```
[the remark this answers is not in the document]
Comment (jadudm) [reply .github/ISSUE_TEMPLATE/story-issue-template-.md]: Why re-incorporated, and brought it closer to BDD along the way.

File: .github/ISSUE_TEMPLATE/story-issue-template-.md (hunk 1 above)
Comment (jadudm) [reply .github/ISSUE_TEMPLATE/story-issue-template-.md:43]: Admittedly optimizing "for me." I don't like having text I have to remove in order to start writing. However, I did add a comment that reminds us that these items need to be verifiable.

File: .github/ISSUE_TEMPLATE/story-issue-template-.md (hunk 2 above)
Comment (jadudm) [reply .github/ISSUE_TEMPLATE/story-issue-template-.md]: Cleaned up these bits.

File: .github/ISSUE_TEMPLATE/story-issue-template-.md (hunk 3 above)
Comment (jadudm) [reply .github/ISSUE_TEMPLATE/story-issue-template-.md]: Removed.

File: .github/ISSUE_TEMPLATE/story-issue-template-.md (hunk 4 above)
Comment (jadudm) [reply .github/ISSUE_TEMPLATE/story-issue-template-.md:34]: See above; I did add a comment, but did not put text "in my way."

File: .github/ISSUE_TEMPLATE/story-issue-template-.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (jadudm) [reply .github/ISSUE_TEMPLATE/story-issue-template-.md]: (Or, no, no particular reason.)

File: .github/ISSUE_TEMPLATE/story-issue-template-.md [hunk 7]
```diff
@@ -8,26 +8,36 @@ assignees: ''
 
 # At a glance
 
-[comment]: # "Begin with a short summary of this ticket, so its intent can be understood at a glance."
+[comment]: # "Begin with a short summary so intent can be understood at a glance."
 
+[comment]: # "As: a person or role"
```
Comment (mogul) [inline .github/ISSUE_TEMPLATE/story-issue-template-.md:13]: I still think you should be going with the "In order to" template here [for all the reasons](https://blog.crisp.se/2014/09/25/david-evans/as-a-i-want-so-that-considered-harmful), but I'll leave it alone.

**Pull request merged**